cql3/statements/alter_keyspace_statement.cc

```diff
@@ -101,6 +101,8 @@ cql3::statements::alter_keyspace_statement::prepare_schema_mutations(query_proce
         auto ks_md_update = _attrs->as_ks_metadata_update(ks_md, tm, feat);
         std::vector<mutation> muts;
         std::vector<sstring> warnings;
+        auto rf_map = _attrs->get_replication_map();
+        rf_map.erase(ks_prop_defs::REPLICATION_FACTOR_KEY);
```

Comment:
but it can apply to a new DC

Reply:
we need to auto-expand it as in `ks_prop_defs`.`prepare_options`

Reply:
A note to myself: whether to support this tag or drop it is under discussion.

Reply:
my comment above was made before the decision to not support replication_factor.  But that said, we shouldn't silently ignore it 
If it's present in the ALTER KEYSPACE statement attrs we should reject it with an error telling the user what to do instead.

Reply:
Now we throw when ALTER KS comes with `replication_factor` tag.